src/app/core/component/comment/comment.component.ts

```diff
@@ -152,7 +152,9 @@ export class CommentComponent implements OnInit {
       .replace(`#ML${this.userId}`, '')
       .replace('#MisskeyLive', '')
       .replace(`https://live.misskey.io/${this.userId}`, '')
-      .replace(`https://live.misskey.io/@${this.userId}`, '');
+      .replace(`https://live.misskey.io/@${this.userId}`, '')
+      .replace(`<https://live.misskey.io/${this.userId}>`, '')
+      .replace(`<https://live.misskey.io/@${this.userId}>`, '');
```

Comment:
```suggestion
      .replace(`<>`, '');
```

にしないと上でURLを置き換えてるので<>になりそうです

Reply:
確かにそうですね。直しました。